Makefile

```diff
@@ -0,0 +1,53 @@
+# Settings
+# --------
+
+export BUILD_DIR:=$(CURDIR)/.build
+
+K_SUBMODULE:=$(BUILD_DIR)/k
+
+.PHONY: all clean deps k-deps distclean clean-submodules kore-exec
+
+all: kore-exec
+
+test: kore-tests k-tests
+
+kore-tests:
+	stack test --pedantic
+
+k-tests: kore-exec k-deps
+	$(MAKE) -C src/main/k/working test
+
+kore-exec: $(BUILD_DIR)/bin/kore-exec
+
+$(BUILD_DIR)/bin/kore-exec:
```

Comment:
Instead of installing `kore-exec` into a custom location, could we get the path from `stack path --local-install-root` instead? That would allow reusing any previous build and `stack clean` would have the desired effect in the `clean` target below.